content/en/docs/ambient/usage/verify-mtls-enabled/index.md

```diff
@@ -0,0 +1,91 @@
+default      ratings-v1-68d5f5486b-b5sbj             10.42.0.6  k3d-k3s-default-agent-0  None     HBONE
+default      reviews-v1-7dc5fc4b46-ndrq9             10.42.1.5  k3d-k3s-default-agent-1  None     HBONE
+default      reviews-v2-6cf45d556b-4k4md             10.42.0.7  k3d-k3s-default-agent-0  None     HBONE
+default      reviews-v3-86cb7d97f8-zxzl4             10.42.1.6  k3d-k3s-default-agent-1  None     HBONE
+{{< /text >}}
+
+Having HBONE configured on your workload doesn't mean your workload will accept reject any plaintext traffic. If you want your workload to reject plaintext traffic, setting a `PeerAuthentication` policy with mTLS mode set to `STRICT` for your workload.
+
+### Validate mTLS from metrics
+
+If you have [installed Prometheus](docs/ops/integrations/prometheus/#installation), you can setup port-forwarding for Prometheus by using the following command:
+
+{{< text syntax=bash >}}
+$ istioctl dashboard prometheus
+{{< /text >}}
+
+View the values for the TCP metrics in the Prometheus browser window. Select Graph. Enter the `istio_tcp_connections_opened_total` metric or `istio_tcp_connections_closed_total` or `istio_tcp_received_bytes_total` or `istio_tcp_sent_bytes_total` and select Execute. The table displayed in the Console tab includes entries similar as below:
+
+{{< text syntax=text >}}
+istio_tcp_connections_opened_total{
+  app="ztunnel", 
+  connection_security_policy="mutual_tls", 
+  destination_principal="spiffe://cluster.local/ns/default/sa/bookinfo-details", 
+  destination_service="details.default.svc.cluster.local", 
+  reporter="source", 
+  request_protocol="tcp", 
+  response_flags="-",
+  source_app="sleep", 
+  source_principal="spiffe://cluster.local/ns/default/sa/sleep", source_workload_namespace="default"
+  ...}
+{{< /text >}}
+
+Validate that the `connection_security_policy` value is set to `mutual_tls` along with the expected source and destination identity information.
+
+### Validate mTLS from logs
+
+You can view either source or destination's ztunnel log to confirm mTLS is enabled, along with peer identities. Below is an example of the source ztunnel's log for a request from the `sleep` service to the `details` service:
+
+```
+2024-08-21T15:32:05.754291Z	info	access	connection complete	src.addr=10.42.0.9:33772 src.workload="sleep-7656cf8794-6lsm4" src.namespace="default" src.identity="spiffe://cluster.local/ns/default/sa/sleep" dst.addr=10.42.0.5:15008 dst.hbone_addr=10.42.0.5:9080 dst.service="details.default.svc.cluster.local" dst.workload="details-v1-857849f66-ft8wx" dst.namespace="default" dst.identity="spiffe://cluster.local/ns/default/sa/bookinfo-details" direction="outbound" bytes_sent=84 bytes_recv=358 duration="15ms"
+```
+
+Refer to the [verifying ztunnel traffic through logs section](docs/ambient/usage/troubleshoot-ztunnel/#verifying-ztunnel-traffic-through-logs) for more details.
```

Comment:
Neither here nor in the linked page do we specifically mention specifically what we want them to look for to validate mTLS is working. We should probably have them check that src/dst identity is populated/correct since this is the most direct tie in to the certs we used.

Reply:
updated!